pkg/katautils/config-settings.go

```diff
@@ -9,6 +9,7 @@
 package katautils
 
 var defaultHypervisorPath = "/usr/bin/qemu-lite-system-x86_64"
+var defaultJailerPath = "/usr/bin/jailer"
```

Comment:
Wow, that's not a great name for a seemingly FC-specific command ;(

Reply:
Agree. Right now it is specific to firecracker. But...

@jodh-intel @devimc @egernst @sboeuf we have a more fundamental problem. In the case of QEMU, we implement functionality that in the case of firecracker is delegated to the jailer. 

However the jailing functionality implemented for QEMU places the runtime itself in the jail vs just QEMU. Some of this is because of limitation in go and not necessarily the way we want it function. This will cause problems for us.

I am trying to see if we can create a cleaner interface that will work for both QEMU and firecracker.

You could envisage creating something similar to jailer created for QEMU so that we work around the go limitations in a cleaner manner.

Currently I am leaning towards disabling the self jailing code as a part of the hypervisor configuration. (i.e if a jailer is present, do not jail the runtime itself). The runtime can create the namespaces, but will not run in it all the time.

Reply:
If we can reach a point where we have a jailer for QEMU, and let's go even further by saying we would have a jailer capable of isolating any process as a simple wrapper exec'ing the given process (reminds me about containers...), we would need to modify the `hypervisor` Go interface defined in virtcontainers so that it would support passing a few extra information about those expected isolation layers.